Makefile

```diff
@@ -11,6 +11,7 @@ build:
 install:
 	install -Dm00755 fin $(DESTDIR)$(PREFIX)/bin/fin
 	install -Dm00644 fin.service $(DESTDIR)$(PREFIX)/lib/systemd/system/fin.service
+	test ! -f /etc/pam.d/display_manager && install -Dm00644 /etc/pam.d/login /etc/pam.d/display_manager || true
```

Comment:
I’m wondering if this isn’t something that the distribution should set up. Is this really something that we should have to worry with creating? It just feels like something that wouldn’t be our issue to be solved, but that’s just my feeling of it. I could certainly be wrong.

Reply:
The application that requires the pam module should be the one that creates the ruleset. 
Using sshd on debian as an example, the config gets packaged up inside of the archive: gethttps://packages.debian.org/buster/amd64/openssh-server/filelist

However, it's not part of the official release by the OpenSSH team: https://ftp.openbsd.org/pub/OpenBSD/OpenSSH/openssh-8.8.tar.gz

So yes, you're probably correct, I can remove the Makefile line. Should I add a troubleshooting section to the README instead, which suggests the placement of the file, alongside the pulseaudio and dbus hints?

Reply:
Let's hear what @andydotxyz has to say about it first :)

Reply:
It's tricky but I also feel like this would be a packaging issue.
Generally opening up auth holes in "make install" seems like taking advantage of the sudo permission IMHO.
Could it be noted in the docs instead? Thanks @taigrr